services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -27,92 +48,161 @@
+  private final DoubleHistogram latencySensorOtel;
+
+  /** retry metrics */
+  private final Sensor errorRetryCountSensor;
+  private final LongCounter retryTriggeredSensorOtel;
+  private final Sensor allowedRetryRequestSensor;
+  private final LongCounter allowedRetryRequestSensorOtel;
+  private final Sensor disallowedRetryRequestSensor;
+  private final LongCounter disallowedRetryRequestSensorOtel;
+  private final Sensor retryDelaySensor;
+  private final DoubleHistogram retryDelaySensorOtel;
+
+  /** retry aborted metrics */
+  private final Sensor delayConstraintAbortedRetryRequest;
+  private final Sensor slowRouteAbortedRetryRequest;
+  private final Sensor retryRouteLimitAbortedRetryRequest;
+  private final Sensor noAvailableReplicaAbortedRetryRequest;
+  private final LongCounter abortedRetrySensorOtel;
+
+  /** key count metrics */
+  private final Sensor keyNumSensor;
+  private final Sensor badRequestKeyCountSensor;
+  private final DoubleHistogram keyCountSensorOtel;
+
+  /** OTel metrics yet to be added */
   private final Sensor requestSizeSensor;
   private final Sensor compressedResponseSizeSensor;
   private final Sensor responseSizeSensor;
-  private final Sensor badRequestSensor;
-  private final Sensor badRequestKeyCountSensor;
   private final Sensor requestThrottledByRouterCapacitySensor;
   private final Sensor decompressionTimeSensor;
   private final Sensor routerResponseWaitingTimeSensor;
   private final Sensor fanoutRequestCountSensor;
   private final Sensor quotaSensor;
   private final Sensor findUnhealthyHostRequestSensor;
-  private final Sensor keyNumSensor;
   // Reflect the real request usage, e.g count each key as an unit of request usage.
   private final Sensor requestUsageSensor;
   private final Sensor requestParsingLatencySensor;
   private final Sensor requestRoutingLatencySensor;
   private final Sensor unAvailableRequestSensor;
-  private final Sensor delayConstraintAbortedRetryRequest;
-  private final Sensor slowRouteAbortedRetryRequest;
-  private final Sensor retryRouteLimitAbortedRetryRequest;
-  private final Sensor noAvailableReplicaAbortedRetryRequest;
   private final Sensor readQuotaUsageSensor;
   private final Sensor inFlightRequestSensor;
   private final AtomicInteger currentInFlightRequest;
   private final Sensor unavailableReplicaStreamingRequestSensor;
-  private final Sensor allowedRetryRequestSensor;
-  private final Sensor disallowedRetryRequestSensor;
-  private final Sensor errorRetryAttemptTriggeredByPendingRequestCheckSensor;
-  private final Sensor retryDelaySensor;
   private final Sensor multiGetFallbackSensor;
   private final Sensor metaStoreShadowReadSensor;
   private Sensor keySizeSensor;
+
+  /** TODO: Need to clarify the usage and add new OTel metrics or add it as a part of existing ones */
+  private final Sensor errorRetryAttemptTriggeredByPendingRequestCheckSensor;
+
   private final String systemStoreName;
+  private final Attributes otelMetricDimensions;
```

Comment:
Can we name this - `defaultMetricDimensions` or `commonDimensions` or something along these lines?